src/webview/src/components/fbaCheckbox.js

```diff
@@ -202,23 +222,79 @@ export default function FBACheckbox(props) {
 
     const backgroundFragments = (
         <React.Fragment>
-            <DialogContentText paragraph>
-                <TextField name='backgroundDescription' onChange={handleValueChanges} margin="dense" id={'backgroundDescription-field-' + props.name}
-                    InputLabelProps={{ shrink: true, }} fullWidth multiline value={values.backgroundDescription}
-                    label='Background'
-                    placeholder='Please enter some background information on this root cause.' />
-            </DialogContentText>
+            <Divider variant="middle" />
+            <Accordion>
+                <AccordionSummary
+                    expandIcon={<ExpandMoreIcon />}
+                    aria-controls="panel1a-content"
+                    id="panel1a-header">
```

Comment:
sometimes react needs separate ids. I didn't really get yet when (but it might be safer to just change them to unique names.)